update_registry.py

```diff
@@ -0,0 +1,124 @@
+import os
```

Comment:
Suggested structure:

1. Wrap each step/action a user would want to take into a function stored in a separate file. 
2. Create a section at bottom to run it if called via running the file directly.  
```python
def update_registry(
"""
Docstring about what it does....vars, etc.
"""

if __name__ == '__main__':
   update_registry(
   ...
```